src/main/java/com/swirlds/recordserver/TokensService.java

```diff
@@ -143,4 +146,121 @@ private void getDefaultMessageHandler(ServerRequest request, ServerResponse resp
                         .build());
         response.send(returnObject.build());
     }
+
+/**********************
+### getTokenById not specifiying a timestamp
+GET http://localhost:8080/api/v1/tokens/107594
+### getTokenById specifiying a timestamp
+GET http://localhost:8080/api/v1/tokens/107594?timestamp=1610640452612903002
+### getTokenById specifiying an earlier timestamp
+GET http://localhost:8080/api/v1/tokens/107594?timestamp=1610640421985772001
+### getTokenById specifiying an earlier timestamp than the passed-in one
+GET http://localhost:8080/api/v1/tokens/107594?timestamp=lt:1610640421985772001
+**********************/
+    private void getTokenById(ServerRequest request, ServerResponse response) {
+        System.out.println("Calling into getTokenById!");
+	
+        final Optional<String> timestampQueryParam = request.queryParams().first("timestamp");
```

Comment:
I am not sure If I am missing it, but I don't see filtering by tokenId

Reply:
I know it had been there -- a call to request.path() used to set a whereClause -- but now I don't see it, either.  I'll recreate it and get back to you.